ansible/roles_ocp_workloads/ocp4_workload_windows_node/files/sshcmd.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+ssh_key_param=''
+if [ ! -z "$SSH_KEY_PATH" ]; then
+    ssh_key_param="-i $SSH_KEY_PATH"
+fi
+
+ingress_host="$(oc get service --all-namespaces -l run=ssh-bastion -o go-template='{{ with (index (index .items 0).status.loadBalancer.ingress 0) }}{{ or .hostname .ip }}{{end}}')"
+ssh $ssh_key_param -t -o StrictHostKeyChecking=no -o ProxyCommand="ssh $ssh_key_param -A -o StrictHostKeyChecking=no -o ServerAliveInterval=30 -W %h:%p core@${ingress_host}" administrator@$1 "powershell"
```

Comment:
I guess there isn't an Ansible Windows Module for this?

Reply:
This script is for the end user so they can SSH into the windows node via a container. I wasn't able to ssh from the bastion server provided from RHPDS. This is more for the user to use if they want to take a look at the windows node.